examples/supertuxkart/Dockerfile

```
@@ -16,45 +16,25 @@
 FROM golang:1.17.2 as go-builder
 WORKDIR /go/src/supertuxkart
 
-RUN apt-get update && apt-get install -y curl tar xz-utils
-
 COPY main.go .
 COPY go.mod .
 RUN go mod tidy
 RUN go build -o wrapper .
 
 # compile stk
-FROM debian:buster as stk-builder
-
-RUN apt update && apt-get install -y build-essential cmake libbluetooth-dev \
-libcurl4-openssl-dev libenet-dev libfreetype6-dev libharfbuzz-dev libfribidi-dev \
-libgl1-mesa-dev libglew-dev libjpeg-dev libogg-dev libopenal-dev libpng-dev \
-libssl-dev libvorbis-dev libxrandr-dev libx11-dev nettle-dev pkg-config zlib1g-dev \
-git subversion
+FROM debian:bullseye
 
-RUN git clone -b allow-ai https://github.com/markmandel/stk-code.git
-RUN svn co https://svn.code.sf.net/p/supertuxkart/code/stk-assets stk-assets
-RUN rm -rf ./stk-code/.git
-RUN rm -rf ./stk-assets/.svn
+RUN apt update && apt-get install -y  wget xz-utils
 
-RUN cd stk-code && mkdir cmake_build && cd cmake_build \
-    && cmake .. -DBUILD_RECORDER=off -DSERVER_ONLY=ON \
-    && make -j$(nproc)
+RUN wget https://github.com/supertuxkart/stk-code/releases/download/1.3/SuperTuxKart-1.3-linux-64bit.tar.xz
```

```suggestion
RUN wget --quiet https://github.com/supertuxkart/stk-code/releases/download/1.3/SuperTuxKart-1.3-linux-64bit.tar.xz && \
tar -xf SuperTuxKart-1.3-linux-64bit.tar.xz && rm -rf SuperTuxKart-1.3-linux-64bit.tar.xz
```
(there delete lines 31 & 32) -- putting this all on one line should shrink the size of the image, as the container won't need a seperate layer for each command.

Also added `--quiet` to remove the huge amount of output wget makes.